.node-version

```diff
@@ -1 +1 @@
-8.15.0
+12.10.0
```

Comment:
updated node to v12 so that I could use named capture groups in RegExp match data